### PR TITLE
Fixes for Flake8 compliance

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: psql2mysql.py [hunk 1]
```diff
@@ -15,11 +15,13 @@
...
 
-import re
-import six
 from oslo_config import cfg
 from oslo_log import log as logging
 from prettytable import PrettyTable
+
+import re
+import six
+
```
Comment (aplanas) [inline psql2mysql.py]: In Python usually this order is used:

* Python imports (like `re`)
* site-package imports (like six, oslo, prettytable or alchemy)
* local imports (modules from the same code)

File: test-psql2mysql.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 p2m = importlib.import_module("psql2mysql")
 
 
-class FakeColumn():
+class FakeColumn(object):
```
Comment (aplanas) [inline test-psql2mysql.py]: only needed in Python2 not in Python3, but I guess that this will run in both, so is OK

File: psql2mysql.py (hunk 1 above)
Comment (jsuchome) [reply psql2mysql.py]: my local `flake8-2.7` complained about the order, but seems like it is too old

**Comment by jsuchome** [comment]
This actually does not seem to be needed as I thought

**Pull request closed without merging**